server/app.js

```diff
@@ -64,6 +64,21 @@ passport.deserializeUser(async (userID, done) => {
   done(null, query.rows[0]);
 });
 
+function verifyAuthenticated(req, res, next) {
+  if (req.isAuthenticated()) {
+    console.log("AUTHENTICATED - ACCESSING ENDPOINT", req.path);
+    next();
+  } else if (req.path.startsWith('/auth')) {
+    console.log("NOT AUTHENTICATED BUT ACCESSING AUTH ENDPOINT - PROCEEDING TO ENDPOINT", req.path);
+    next();
+  } else {
+    console.log("ERROR 404: NOT AUTHENTICATED - CANNOT ACCESS ENDPOINT", req.path);
+    res.status(404).send();
+  }
+}
```

Comment:
Change status to 403 (unauthorized). 404 is (page not found)
Also, remove console.logs

Reply:
feel free to keep these console.log() statements too!